boards/adafruit/feather_rp2040/adafruit_feather_rp2040_defconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable UART driver
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable reset by default
+CONFIG_RESET=y
+
```

Comment:
```suggestion
```

Reply:
What's the reason for dropping this config value?

Reply:
No idea specifically; it was part of `zephyr/boards/adafruit/qt_py_rp2040/adafruit_qt_py_rp2040_defconfig` which I duplicated as the starting point for this board.
Though if it ought to be removed there too...

Reply:
yes it should be

Reply:
Fixed, at least as to the feather RP2040. All of these formatting fixes should probably also be applied to qt_py_rp2040, though I'll leave that to someone else.

Reply:
After much headache, I've determined that this is actually a dependency of the four failing twister CI tests:

- `drivers.led_strip.build`
- `drivers.adc`
- `drivers.counter.basic_api`
- `drivers.dma.chan_blen_transfer`

I have genuinely no idea _why_ removing this would be inducing these failures, but that's a matter for some other PR; for now I've re-added this defconfig in 92279e70f42.

Reply:
@soburi think something is a bit wrong here, you shouldn't need the reset API to use an adc, that makes no sense

Reply:
hmm wow do I feel like an idiot, got mixed up with `REBOOT`... my bad